binds/python/bind_immutable.cpp

```diff
@@ -53,6 +53,10 @@ static void bind_immutable_module(py::module &m) {
                 return py::array(static_cast<py::ssize_t>(morpho->points().size()), morpho->points().data());
             },
             "Returns a list with all points from all sections")
+        .def_property_readonly("section_offsets", [](morphio::Morphology* morpho){
```

Comment:
I am not sure, but I think pybind11 can accepts lambda taking `const MyClass&` in parameter. It would be safer here:
1. there is no need for pointer because the object is always there
1. the instance should not be modifiable

Reply:
Cool, yes it works.